axis.go

```diff
@@ -361,7 +361,12 @@ func (DefaultTicks) Ticks(min, max float64) (ticks []Tick) {
 	val := math.Floor(min/majorDelta) * majorDelta
 	for val <= max {
 		if val >= min && val <= max {
-			ticks = append(ticks, Tick{Value: val, Label: fmt.Sprintf("%g", float32(val))})
+			round := float32(val)
+			if val < 1 && val > -1 {
+				round = float32(val + 1)
+				round -= 1
+			}
+			ticks = append(ticks, Tick{Value: val, Label: fmt.Sprintf("%g", round)})
```

Comment:
I don't understand your remark about float64(round). I was suggesting using the more-accurate, float32 for the actual tick location in addition to the string value. If the float32 is more accurate why not put the tick mark there? (And I agree that the error is so small that it won't change the display in any meaningful way, but I think that it's confusing to use different values for both if one's more accurate.)

Don't worry about the churn, this would be a very minor addition.